.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+
+node_js:
+  - 4.0
+  - 4
+  - 5
+
+sudo: false
```

Comment:
Relevant blog post : http://blog.travis-ci.com/2015-03-31-docker-default-on-the-way/